src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -191,10 +194,12 @@ class RunWorkflowTests : SeleniumTest()
         createButton.click()
         wait.until(ExpectedConditions.presenceOfElementLocated(By.id("git-branch")))
 
-        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isEqualTo(1)
+        // We generally expect one git commit option in demo orderly before refresh, but have found that this can be two commits on Buildkite run. This may be related to having run a workflow in another test
+        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isIn(listOf(1, 2))
 
         val refreshButton = driver.findElement(By.id("git-refresh-btn"))
         refreshButton.click()
+        assertThat(driver.findElements(By.cssSelector("div[class*='error-message']")).count()).isEqualTo(0)
```

Comment:
This should have been sufficient (or even just `.text-danger`, but adding a new class is obviously more robust!):
```suggestion
        assertThat(driver.findElements(By.cssSelector(".error-message")).count()).isEqualTo(0)
```